angular/projects/admin-nrpti/src/app/mines/mines-add-edit/mines-add-edit.component.scss

```diff
@@ -1 +1,7 @@
 @import 'assets/styles/components/mine.scss';
+
+.summary-label {
+  font-size: small !important;
+  font-style: italic;
+  color: gray;
```

Comment:
There are different grays in the theme file. Might want to pull one of those. There might even be one specific for text IIRC.

Reply:
Swapped out the colour to use the theme `$font-gray`